README.md

```diff
@@ -282,7 +187,3 @@ I would like to thank...
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] In an attempt to make this ReadMe personalized as the previous reviewer suggested, kindly remove this part of your ReadMe as it is not important to your users. It is more professional to keep files that do not add any benefit to our users. This majorly is a guide that explains what this `License` section is about, and it also directs you on how to use or choose from the many types of licenses that exist.

Reply:
i did remove that 
i think youre looking at the wrong branch maybe